tests/unit/table/test_convert_column.py

```diff
@@ -27,7 +28,7 @@
 source_cols: List[TensorColumn] = []
 output_col_types: List[Type] = []
 
-if cp:
+if cp and HAS_GPU:
```

Comment:
Is there a way we could put the `HAS_GPU` in `merlin.core.compat` so we don't have to repeat it in so many places?

Reply:
I think so? Can we import CuPy successfully if we don't have a GPU?

Reply:
that depends on the extent of what "don't have a GPU" means

If CuPy or cudf fail to find relevant cuda drivers they raise an ImportError which we catch in compat. and `cp` will be None. However, if we set `CUDA_VISIBLE_DEVICES=` in an environment that has the relevant cuda drivers and libs available then the import is successful, `cp` will have the value of the cupy module, but if you try to use it e.g. cupy.array(...), then you get a `CUDARuntimeError`

Reply:
Is it possible this is a misuse of CUDA_VISIBLE_DEVICES and not a scenario we would like to support.

Reply:
In the latter case, is `HAS_GPU` false? If so, then seems like we might be able to use it in compat